.prettierrc

```diff
@@ -1,4 +1,5 @@
 {
   "printWidth": 100,
-  "singleQuote": true
-}
\ No newline at end of file
+  "singleQuote": true,
+  "trailingComma": "es5"
```

Comment:
why this change?

Reply:
Because it was really inconsistent, test files and other configs all had trailing commas, and the ones that prettier handled, did not. Also, it [has some advantages](https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8) WDYT?

Reply:
Go team trailing commas!